srcpkgs/yosys/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'yosys'
+pkgname=yosys
+version=0.8
+revision=1
+wrksrc=${pkgname}-${pkgname}-${version}
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="python3 bison flex pkg-config clang git tcl"
+makedepends="tcl-devel readline-devel libffi-devel"
+depends="graphviz xdot"
+short_desc="Verilog RTL Synthesis Suite"
+maintainer="Jasper Chan <jasperchan515@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://www.clifford.at/yosys/about.html"
+distfiles="https://github.com/cliffordwolf/${pkgname}/archive/${pkgname}-${version}.tar.gz"
```

Comment:
please write out the pkgname instead of using a variable

Reply:
Done